Assets/00_Content/Scripts/Vikings/PassiveVikingState.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Vikings {
+	public class PassiveVikingState : VikingState {
+		public PassiveVikingState(Viking viking) : base(viking) {
+		}
+
+		public override VikingState Update() {
+			viking.Stats.Decline();
+
+			if (viking.Stats.Mood < 50)
```

Comment:
Just so I understand this, does a vikings desire only change when their mood is above 50?

Reply:
Not really, this causes the viking to desire something once its mood drops below 50. This is one of these hardcoded values to simulate player interaction. This changes the state to desiring something which has not been fully implemented yet.